client/src/elm/Pages/MessagingCenter/View.elm

```diff
@@ -80,21 +80,21 @@ view language currentTime nurseId nurse db model =
                                         Dict.values surveys
                                             |> List.sortWith (sortByDateDesc .dateMeasured)
 
-                                    runMonthlySurvery =
-                                        List.filter (.surveyType >> (==) ResilienceSurveyMonthly) surveysSorted
+                                    runQuarterlySurvey =
+                                        List.filter (.surveyType >> (==) ResilienceSurveyQuarterly) surveysSorted
                                             |> List.head
                                             |> Maybe.map
                                                 (\survey ->
-                                                    -- Run monthly survey if one month has passed since
-                                                    -- last monthly survey was completed.
-                                                    Date.diff Months survey.dateMeasured currentDate >= 1
+                                                    -- Run a 3 months survey if 3 month has passed since
+                                                    -- last survey was completed.
+                                                    Date.diff Months survey.dateMeasured currentDate >= 3
                                                 )
-                                            -- No monthly survey were performed, so we need to run one if
-                                            -- one month has passed since program has started.
-                                            |> Maybe.withDefault (Date.diff Months programStartDate currentDate >= 1)
+                                            -- We need to run the survey if
+                                            -- it is the beginning of the Program(Baseline).
+                                            |> Maybe.withDefault (Date.diff Months programStartDate currentDate == 0)
```

Comment:
This seems wrong

Reply:
Again, this is wrong. As it's set now, there will be no survey unless nurse logged in on same day program was activated.

Reply:
@Diama1 ^^